spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    sequence(:email) { |i| "email#{i}@email.com"}
```

Comment:
`sequence(:email) { |index| Faker::Internet.email.sub(/\@/, "#{index}_@") }` for more randomized